Description: Man pages typos
 Man pages typos found by lintian
Author: Gennaro Oliva <oliva.g@na.icar.cnr.it>
Last-Update: <2013-10-01>

--- slurm-llnl-2.6.3.orig/doc/man/man1/sh5util.1
+++ slurm-llnl-2.6.3/doc/man/man1/sh5util.1
@@ -158,16 +158,16 @@ Write_Megabytes
 .TP
 Merge node-step files (as part of a sbatch script)
 .LP
-sbatch -n1 -d$SLURM_JOB_ID --wrap="sh5util --savefiles -j $SLURM_JOB_ID"
+sbatch \-n1 \-d$SLURM_JOB_ID \-\-wrap="sh5util \-\-savefiles \-j $SLURM_JOB_ID"
 
 .TP
 Extract all task data from a node
 .LP
-sh5util -j 42 -N snowflake01 --level=Node:TimeSeries --series=Tasks
+sh5util \-j 42 \-N snowflake01 \-\-level=Node:TimeSeries \-\-series=Tasks
 
 .TP
 Extract all energy data
-sh5util -j 42 --series=Energy --data=power
+sh5util \-j 42 \-\-series=Energy \-\-data=power
 
 .SH "COPYING"
 Copyright (C) 2013 Bull.
--- slurm-llnl-2.6.2.orig/doc/man/man5/acct_gather.conf.5
+++ slurm-llnl-2.6.2/doc/man/man5/acct_gather.conf.5
@@ -1,4 +1,4 @@
-.TH "acct_gather.conf" "3" "May 2013" "acct_gather.conf 1.0" "Slurm acct_gather.configuration file"
+.TH "acct_gather.conf" "5" "May 2013" "acct_gather.conf 1.0" "Slurm acct_gather.configuration file"
 
 .SH "NAME"
 acct_gather.conf \- Slurm configuration file for the acct_gather plugins
--- slurm-llnl-2.6.2.orig/doc/man/man5/cray.conf.5
+++ slurm-llnl-2.6.2/doc/man/man5/cray.conf.5
@@ -23,7 +23,7 @@ of the command "scontrol reconfigure" un
 The configuration parameters available include:
 
 .TP
-\fBAlpsEngine\R=<engine_version>
+\fBAlpsEngine\fR=<engine_version>
 Communication protocol version number to be used between Slurm and ALPS/BASIL.
 The default value is BASIL's response to the ENGINE query.
 Use with caution: Changes in ALPS communications which are not recognized
--- slurm-llnl-2.6.2.orig/doc/man/man5/slurm.conf.5
+++ slurm-llnl-2.6.2/doc/man/man5/slurm.conf.5
@@ -199,7 +199,7 @@ execution of the command "sudo modprobe
 \fBAcctGatherInfinibandType\fR
 Identifies the plugin to be used for infiniband network traffic accounting.
 The plugin is activated only when profiling on hdf5 files is activated and
-the user asks for network data collection for jobs through --profile=Network
+the user asks for network data collection for jobs through \-\-profile=Network
 (or =All). The collection of network traffic data takes place on node level,
 hence only in case of exclusive job allocation the collected values will
 reflect the jobs real traffic. All network traffic data are logged on hdf5 files
--- slurm-llnl-2.6.2.orig/doc/man/man5/slurmdbd.conf.5
+++ slurm-llnl-2.6.2/doc/man/man5/slurmdbd.conf.5
@@ -1,4 +1,4 @@
-.TH "slurmdbd.conf" "28" "Feb 2013" "slurmdbd.conf 2.6" "Slurm configuration file"
+.TH "slurmdbd.conf" "5" "Feb 2013" "slurmdbd.conf 2.6" "Slurm configuration file"
 .SH "NAME"
 slurmdbd.conf \- Slurm Database Daemon (SlurmDBD) configuration file
 
